src/AutoRest.CSharp/Common/Input/InputTypes/BytesKnownEncoding.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace AutoRest.CSharp.Common.Input;
+
+internal static class BytesKnownEncoding
+{
+    public const string Base64 = "base64";
+    public const string Base64Url = "base64url";
```

Comment:
Shall we be consistent to use enum here?

Reply:
or maybe in TCGC this is defined as different type?

Reply:
No we cannot use enum here.
the type bytes is different from datetime and duration.
In TCGC, datetime and duration are standalone types, paralell with built-in types.
But the type bytes is a variant of built-in type. For this reason, the `encode` on built-in type (our InputPrimitiveType) must be `string` instead of enum.
Otherwise, please think, how should we name the enum? This encode should be shared with all types in the primitive types, it should be open to expand.